recipes/bredis/all/test_package/test_package.cpp

```diff
@@ -0,0 +1,30 @@
+#include <cstdlib>
+#include <iostream>
+
+#include "boost/asio.hpp"
+#include "bredis/MarkerHelpers.hpp"
+#include "bredis/Connection.hpp"
+
+namespace r = bredis;
+namespace asio = boost::asio;
+
+static void dummy() {
+  using socket_t = asio::ip::tcp::socket;
+  using next_layer_t = socket_t;
+
+  asio::io_context io_service;
+  auto             ip_address = asio::ip::make_address("127.0.0.1");
+  auto             port       = boost::lexical_cast<std::uint16_t>("6379");
+  std::cout << "connecting to " << ip_address << "\n";
+  asio::ip::tcp::endpoint end_point(ip_address, port);
+  socket_t                socket(io_service, end_point.protocol());
+  socket.connect(end_point);
```

Comment:
Let's avoid reaching the network layer for test packages if possible :)

Reply:
@RubenRBS 
These lines are in the dummy function which has been never called.
Is it not acceptable?

Reply:
We need funtions whose implementation lives in a .cpp to ensure those work too, and they need to be called for the compiler not to simply remove dead code, else it might just ignore it even if it's wrong :)

Reply:
Fixed.